src/commands/utility/weather.js

```diff
@@ -0,0 +1,76 @@
+const { CanvasTemplates, CommandStructures, SwitchbladeEmbed, Constants } = require('../../')
+const { Command, CommandParameters, StringParameter } = CommandStructures
+const { Attachment } = require('discord.js')
+const moment = require('moment-timezone')
+
+const icons = [ 'clear-day',
+  'clear-night',
+  'cloudy',
+  'fog',
+  'partly-cloudy-day',
+  'partly-cloudy-night',
+  'rain',
+  'snow',
+  'thunderstorm',
+  'wind' ]
+
+module.exports = class Weather extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'weather'
+    this.category = 'utility'
+
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ full: true, required: true, missingError: 'commands:weather.noCity' })
+    )
+  }
+
+  async run ({ t, author, channel, language }, address) {
+    moment.locale(language)
+    channel.startTyping()
+    const city = await this.client.apis.gmaps.searchCity(address)
+    if (city) {
+      const cityLocation = city.geometry.location
+      const weatherData = await this.client.apis.darksky.getForecast(cityLocation.lat, cityLocation.lng, { lang: language.split('-')[0], units: 'ca' })
+      const currently = weatherData.currently
+      const daily = weatherData.daily.data
```

Comment:
`const { currently, daily: { data: daily }, timezone } = await this.client.apis.darksky...`
(then replace [`weatherData.timezone`](https://github.com/SwitchbladeBot/switchblade/pull/498/files#diff-ec8eccb21973782ce759bff0dde79fd4R51) with just `timezone`)